api/tests/unit/domain/services/certification-service_test.js

```diff
@@ -295,4 +306,61 @@ describe('Unit | Service | Certification Service', function() {
 
     });
   });
+
+  describe('#createNewCertification', () => {
+
+    let clock;
+    let sandbox;
+
+    const certificationCourse = { id: 'newlyCreatedCertificationCourse' };
+    const certificationCourseWithChallengesNumber = { id: 'newlyCreatedCertificationCourse', nbChallenges: 3 };
+    const userProfile = [{ id: 'competence1', challenges: [] }];
+
+    beforeEach(() => {
+      clock = sinon.useFakeTimers(new Date('2018-02-04T01:00:00.000+01:00'));
+
+      sandbox = sinon.sandbox.create();
+      sandbox.stub(certificationCourseRepository, 'save').resolves(certificationCourse);
+      sandbox.stub(userService, 'getProfileToCertify').resolves(userProfile);
+      sandbox.stub(certificationChallengesService, 'saveChallenges').resolves(certificationCourseWithChallengesNumber);
+
+    });
+
+    afterEach(() => {
+      clock.restore();
+      sandbox.restore();
+    });
+
+    it('should create the certification course with status "started"', function() {
+      // given
+      const userId = 12345;
+
+      // when
+      const promise = certificationService.createNewCertification(userId);
+
+      // then
+      return promise.then((newCertification) => {
+        sinon.assert.calledOnce(certificationCourseRepository.save);
+        sinon.assert.calledWith(certificationCourseRepository.save, { userId: userId, status: 'started' });
```

Comment:
On pourrait basculer l'écriture de ces tests là en utilisant les expect()

`expect(certificationCourseRepository.save).to.have.been.calledWith({ userId: userId, status: 'started' });`

Reply:
Anéfé !
J'ai bêtement copié/collé un test que j'avais trouvé :)